main.js

```diff
@@ -9,15 +11,44 @@ let game = {
     cardsTurn: [],
     // Stores the number of clicks in the current turn 0 - 2
     clicks: 0,
-    turn: 0
+    turn: 0,
+    timer: 0,
+    timerSwitch: 0,
+    gameFinished: false,
+    timerObject: function (){setInterval(function () {
```

Comment:
Remove spaces between function and params, and add between params and curly brace